### PR TITLE
feat: 0.55 data migration script

### PR DESCRIPTION
**Pull request by nityanandagohain**
* It takes a timestamp as an input.
* tries to paginate using timestamp.
   * if there is timestamp collision for that ts then limit offset is used.
 
 
 This logic lies in the  processBatchesOfLogs function

### CODE REVIEW

**Comment by srikanthccv** [comment]
Alternatively, forget about the pagination and write in batches of 5-minute moving windows?

**Comment by nityanandagohain** [comment]
Do you mean bringing all the data for 5 mins in a single call and then process it ?

**Comment by nityanandagohain** [comment]
or something like, paginating for the five 5mins using limit offset ?

**Comment by nityanandagohain** [comment]
Yeah it makes sense, 

Will keep the time range configurable so that if someone is sending very high amount of data they might tweak it, but the solution will work. Thanks for the suggestion 👍

File: migration-0.55-data/main.go [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+
+	flag.Parse()
+
+	sourceConn, err := connect(*hostFlag, *portFlag, *userNameFlag, *passwordFlag, *sourceDbName)
+	if err != nil {
+		zap.S().Fatal("Error while connecting to clickhouse", zap.Error(err))
+	}
+	defer sourceConn.Close()
+
+	destConn, err := connect("localhost", *portFlag, *userNameFlag, *passwordFlag, *destDbName)
```
Comment (srikanthccv) [inline migration-0.55-data/main.go:338]:
```suggestion
	destConn, err := connect(*hostFlag, *portFlag, *userNameFlag, *passwordFlag, *destDbName)
```
?

**Review by srikanthccv — DISMISSED** [review]

File: migration-0.55-data/main.go (hunk 1 above)
Comment (nityanandagohain) [reply migration-0.55-data/main.go]: Ahh my bad,  updated the flags for dest connection where user might use a different clickhouse cluster.

**Review by srikanthccv — APPROVED** [review]

**Pull request merged**